build-package

```diff
@@ -37,12 +37,9 @@ install -Dm660 \
 
 # Install systemd units
 install -Dm644 "$STARTDIR/systemd/system.service" \
-	"$DESTDIR/lib/systemd/system/thelounge.service"
+	"$DESTDIR/usr/lib/systemd/system/thelounge.service"
```

Comment:
This is incorrect: on Debian/Ubuntu, the folder is indeed `/lib/systemd/system` contrary to Arch. This fails to install in `/usr/lib/systemd/system`.

Reply:
Also note that this is only for system. For user units it's indeed `/usr/lib` because Debian

Reply:
You gotta be kidding me... Thanks!
Sorry I'm not used to debianisms.